packages/core/tests/helpers.ts

```diff
@@ -664,3 +664,26 @@ export async function setupProofsTest(faberName: string, aliceName: string, auto
     aliceReplay,
   }
 }
+
+export async function setupSecondCredential(faberAgent: Agent, aliceAgent: Agent, faberConnection: ConnectionRecord) {
+  const { definition } = await prepareForIssuance(faberAgent, ['name', 'age'])
+  const secCredDefId = definition.id
+
+  const credentialPreview = CredentialPreview.fromRecord({
+    name: 'John',
+    age: '99',
+  })
+
+  await issueCredential({
+    issuerAgent: faberAgent,
+    issuerConnectionId: faberConnection.id,
+    holderAgent: aliceAgent,
+    credentialTemplate: {
+      credentialDefinitionId: secCredDefId,
+      comment: 'some comment about credential',
+      preview: credentialPreview,
+    },
+  })
+
+  return { secCredDefId: definition.id }
```

Comment:
We generally do not abbriviate variables

Reply:
I actually took this from the function above
https://github.com/hyperledger/aries-framework-javascript/pull/640/files/da58c95310b499ea2ae3a4b84c6d728fb916e487#diff-ea192c0081e4b56d18cea33ca16a6b4d7cb2ccc5b4b23c183ba511843e30b60bL659

should I change it to return just the `definition.id` and take a string as a return type?

Reply:
Yes, I think that is even better.